src/module/CommentTest/getComment.test.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
[OPTIONAL] Unfortunately, your tests were not built properly. You should write tests for the counter functions. But you are creating some functions and testing them which is not useful for testing the functionality of the app. 
<img width="1072" alt="Screenshot 2023-03-25 at 2 56 22 AM" src="https://user-images.githubusercontent.com/25887253/227664534-59265fda-26e7-41d5-80a2-dc1f36af3ffc.png">